DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Please also move the data connector and logo files into the solutions folder.

/Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json
/Solutions/Flare Systems Firework/Data Connectors/Logo/FlareSystems.svg
Sample data can remain in its current location.

Reply:
Done!